larky/src/main/java/com/verygood/security/larky/parser/LarkyEvaluator.java

```diff
@@ -181,6 +184,19 @@ class LarkyLoader implements StarlarkThread.Loader {
     @Nullable
     @Override
     public Module load(String moduleToLoad) {
+      String name = ResourceContentStarFile.getModulePath(moduleToLoad);
+
+      if (TEST_ONLY_MODULE_NAMES.contains(name)) {
+          throw new RuntimeException(
+                  String.format(
+                  "The module '%s' is only available in test mode.%n" +
+                  "You attempted to import a testing module in a normal Larky script.%n" +
```

Comment:
Minor comment, "normal" is to vague for this msg.

```suggestion
                  "You attempted to import a testing module in a production Larky script.%n" +
```